Makefile

```diff
@@ -1,6 +1,6 @@
 libbitarr.a: 
-	cd BitArray && $(MAKE)
-	make -C BitArray -f Makefile
+	cd bp/BitArray && $(MAKE)
+	make -C bp/BitArray -f Makefile
```

Comment:
@fedarko, any chance you can explain why this needed to be added and why building the project worked before these changes? Maybe I'm missing something but nothing else changed, right?

Reply:
> why building the project worked before these changes

I'm not 100% sure on how the build process for iow works. My understanding is that building the project succeeded because it didn't just call `make build` or `make libbitarr.a`: instead, the setup script calls

https://github.com/biocore/improved-octo-waddle/blob/2f92d78d5f1f68168c3ba1adf6979603a257a592/setup.py#L41-L46

The `make -C bitarr` thing tells `make` to move to the directory referred to by `bitarr`(`bp/BitArray`) before running this command. Although I feel like this should fail for another reason, because then `make libbitarr.a` tries to cd into `BitArray` again, and this directory does not exist (we're already in `BitArray`, and [there shouldn't be another `BitArray` directory underneath it](https://github.com/noporpoise/BitArray/tree/f444eb45ab3cac67d00089c50f6bb42729838db8)). So... I got nothing?

> why this needed to be added

The main reason I added this fix in this PR was not because of the build but because it broke `make test`. I looked into it, because the GitHub Actions build was passing before -- the reason it was passing before this fix is because **`make test` is not actually how GH Actions runs its tests**. Instead, GH Actions just uses `pytest`:

https://github.com/biocore/improved-octo-waddle/blob/2f92d78d5f1f68168c3ba1adf6979603a257a592/.github/workflows/python-package-conda.yml#L40-L43

This is separate from how `make test` works:

https://github.com/biocore/improved-octo-waddle/blob/2f92d78d5f1f68168c3ba1adf6979603a257a592/Makefile#L1-L11

It'd probably be ideal to combine these into a single test-running command, but I'm not sure how to do that (and that is likely out of scope for this PR).

----

### TLDR

If you'd like, I can remove this change for now -- just to avoid increasing weird issues with installation. As a separate issue, I think it'd make sense to change `make test` to reflect the GitHub Actions stuff (i.e. just running `pytest`)?